src/builder.rs

```diff
@@ -87,13 +91,15 @@ impl EntityBuilder {
             .write_unaligned(component);
     }
 
+    /// Builds the components into an entity that can be inserted into a world.
     pub fn build(self) -> BuiltEntity<'static> {
         BuiltEntity {
             builder: CowMut::Owned(self),
             written: false,
         }
     }
 
+    /// This one seems redudent, since spawning clears the internal components.
```

Comment:
`build_one` can be used to re-use the `EntityBuilder`'s allocations as an optimization.